c/ldv-linux-3.4-simple/43_1a_cilled_true-unreach-call_ok_nondet_linux-43_1a-drivers--platform--x86--topstar-laptop.ko-ldv_main0_sequence_infinite_withcheck_stateful.cil.out.c

```diff
@@ -3659,6 +3662,24 @@ void ldv_blast_assert(void)
   ERROR: __VERIFIER_error();
 }
 }
+
+void* ldv_successfull_alloc(size_t size)
+{
+  void *res ;
+  long tmp ;
+
+  res = malloc(size);
+  __VERIFIER_assume((unsigned long )res != (unsigned long )((void *)0));
```

Comment:
This is probably redundant as `malloc` should never fail (as well as the assume on line 3801), but it can not harm. I guess this code is somehow generated?

Reply:
You are right, the code is generated, and we use the pattern not to refer to sv-comp rules. Particularly, for sv-benchmarks the assumption is redundant, but it may be useful if benchmarks will be used stand-alone, without any preliminary requirements.